src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/JS/WasmTestCommand.cs

```diff
@@ -31,6 +32,34 @@ public WasmTestCommand() : base("test", true, CommandHelp)
         {
         }
 
+        static readonly string[] s_extensions = { ".cmd", ".exe" };
+
+        private static string GetFullPathTo(string engineBinary)
+        {
+            if (Path.IsPathRooted(engineBinary))
```

Comment:
Should we also check `File.Exists(engineBinary)` for cases where the binary is in the current working directory or we sent a relative path to the binary?
(and root the path in that case and return)